### PR TITLE
Add sixMontly report in example

### PR DESCRIPTION
**Pull request by hakifran**
- Add sixMontly report in example

- change the way sixMontly period is displayed

- display the sixMonthly invoice

- add spec for DatePeriods class

blsq report components

rbf reporting

### CODE REVIEW

File: src/components/invoices/InvoiceLink.js [hunk 1]
```diff
@@ -1,90 +1,110 @@
...
           >
             {invoiceType.name}
           </Button>
         )}
+        {invoiceType.frequency === 'sixMonthly' && (
+          <React.Fragment>
+            <span className={this.props.classes.buttonLike}>
+              {invoiceType.name}
+            </span>
+            {DatePeriods.splitYearQuarter(this.props.period, 'sixMonthly').map(
```
Comment (mestachs) [inline src/components/invoices/InvoiceLink.js]: I'd like too keep the "used" api of DatePeriods to
 - DatePeriods.split(period,  frequency)
 - DatePeriods.displayName(period, format)
all other functions should stay "private/unused"
and adapt the spec of DatePeriods if necessary.

**Comment by mestachs** [comment]
@hakifran can you investigate why your "prettier" don't take the same config as mine ?

File: src/support/Dhis2.js [hunk 2]
```diff
@@ -31,321 +31,321 @@ class Dhis2 {
...
-    const quarter = DatePeriods.split(period, "quarterly")[0].slice(5, 6);
+    const year = period.slice(0, 4)
+    const quarter = DatePeriods.split(period, 'quarterly')[0].slice(5, 6)
 
     return {
       orgUnit: orgUnits.filter(orgUnit => orgUnit.id === orgUnitId)[0],
       orgUnits: orgUnits,
       period: period,
       quarterPeriod: period,
       quarterPeriods: DatePeriods.split(period, invoiceType.frequency),
```
Comment (mestachs) [inline src/support/Dhis2.js]: quarterPeriods should be dhis2 periods quarters

Comment (hakifran) [reply src/support/Dhis2.js]: @mestachs I have fixed the "prettier" configuration in atom
selection_002

Comment (mestachs) [reply src/support/Dhis2.js]: IMO src/support/Dhis2.js
line 302 : quarterPeriods: DatePeriods.split(period, "quarterly"),

Comment (mestachs) [reply src/support/Dhis2.js]: Can you prettyfy this PR with the new settings ?

**Pull request merged**